src/components/common/Onboarding/wizardSteps/StepCreateUser/validation.ts

```diff
@@ -18,7 +13,7 @@ export const MAX_USERNAME_LENGTH = 30;
  * [\w-.] can include upper case, lower case, numerals, underscore, hyphen or period
  * {0,254}$ match the preceding set at least 0 and at most 254 times from the end (so excluding the first char)
  */
-const USERNAME_REGEX = /^[A-Za-z0-9][\w-.]{0,254}$/;
+export const USERNAME_REGEX = /^[A-Za-z0-9][\w-.]{0,254}$/;
```

Comment:
Are we still allowing 255 char length usernames? I remember @arrenv saying they were capped to smaller length

Reply:
we are capping at a smaller however the length validation happens in the `Input` component now and not as part of the validation, the regex has been updated to reflex this